README.md

```diff
@@ -70,17 +70,17 @@ Then a request path like:
 
 will fail because the `sign` parameter is not present.
 
-**The HMAC-SHA256 hash is created by taking the URL path (including the leading /), the request parameters (alphabetically-sorted and concatenated with & into a string). The hash is then base64url-encoded.**
+**The HMAC-SHA256 hash is created by taking the URL path (excluding the leading /), the request parameters (alphabetically-sorted and concatenated with & into a string). The hash is then base64url-encoded.**
```

Comment:
That’s weird because it breaks the `imaginary` lib: https://github.com/h2non/imaginary?tab=readme-ov-file#url-signature.

I think we should maintain this contract by fixing this lib and making a major version release.

Can you add a test that shows the bug?

Reply:
Is there a reason we want to stay compliant with `h2non/imaginary` ? I don't see any mention of that lib in the `plug_image_processing` project. But yeah I don't mind changing this lib to keep it. I'll add a test for it !

Reply:
I've added a test that demonstrate that the "/" is excluded. I'll change the code (and the test) after you give me a 👍 that we want that !

Reply:
Looks like it’s in the repository’s description and tags only:

![](https://github.com/mirego/plug_image_processing/assets/11348/ed763989-fc73-4029-a683-bd58dce425f5)

But we should definitely make it more obvious in `README.md` 😁